docs/protocol-design/work.md

```diff
@@ -14,16 +14,51 @@ Existing content:
 
 ---
 
-## Spam resistance focus
+## Spam resistance
+
+!!! quote ""
+	Proof-of-Work (PoW) is used as a method to deter spam transactions on the network. It is **not** used in the consensus mechanism.
+
+A spam transaction is a block broadcasted with the intention of saturating the network, reducing its available throughput for other network participants, or increasing the size of the ledger. Participants can compute the required PoW in the order of seconds. The cost of spamming the network increases linearly with the number of spam transactions, thus reducing the impact of spam transactions from theoretically infinite to a manageable amount.
 
 ---
 
-## PoW algorithm details
+## PoW in state blocks
+
+Every [state block](./blocks) includes a work field that must be correctly populated. A valid proof is obtained by randomly guessing a nonce such that:
```

Comment:
link doesn't work, should be `/protocol-design/blocks`

Reply:
Oh oops, I wanted to use `../blocks` instead, which works. Then it's not dependent on the directory above it. Do you find that a reasonable approach?